linode/manager
# DO NOT MERGE: Websockets Events POC

### Purpose

Use Websockets to update UI

### Notes
* When an event comes down from websockets, it is stores in Redux state
* Each component will be responsible for updating it's own individual state if an appropriate event exists in store
   * In this case, only Linodes Landing is using this feature
* Events polling has been turned off for the sake of testing

### To Test
1. Make sure your `env` variables are pointing at your local environment
2. Ask me or Will Smith for the username and password to login
3. Create a new Linode
4. Use a SQL client to update the Linode boot status
5. Notice the UI change in Manager

### Concerns

When exactly do we stop polling on a component? For example, if we get an `linode_boot` event from websockets and make staggered `getLinode` requests, do we stop polling? What if the user has booted the Linode and then taken some other action on it?
